README.md

```diff
@@ -8,33 +8,65 @@
 A plugin bringing remote execution to the Foreman, completing the config 
 management functionality with remote management functionality.
 
+* Website: [theforeman.org](http://theforeman.org)
+* ServerFault tag: [Foreman](http://serverfault.com/questions/tagged/foreman)
+* Issues: [foreman_remote_execution Redmine](http://projects.theforeman.org/projects/foreman_remote_execution/issues)
+* Wiki: [Foreman wiki](http://projects.theforeman.org/projects/foreman/wiki/About)
+* Community and support: [#theforeman](https://kiwiirc.com/client/irc.freenode.net/?#theforeman) for general support, [#theforeman-dev](https://kiwiirc.com/client/irc.freenode.net/?#theforeman-dev) for development chat in [Freenode](irc.freenode.net)
+* Mailing lists:
+    * [foreman-users](https://groups.google.com/forum/?fromgroups#!forum/foreman-users)
+    * [foreman-dev](https://groups.google.com/forum/?fromgroups#!forum/foreman-dev)
```

Comment:
I guess, I feel like there's a lot of duplicate data here and we're better off redirecting users to the web site.  Maybe some other support avenue will open up, and it wouldn't be in this README, it's not really on our radar to keep this in sync.